src/shared-components/Button/Button.types.ts

```
@@ -26,10 +26,8 @@ export type ButtonVariant =
  * Props for the shared Button wrapper.
  * Extends react-bootstrap Button props and adds loading, icon, and layout helpers.
  */
-export interface InterfaceButtonProps extends Omit<
-  BootstrapButtonProps,
-  'size' | 'variant'
-> {
+export interface InterfaceButtonProps
+  extends Omit<BootstrapButtonProps, 'size' | 'variant'> {
```

ESLint

[error] Replace `⏎··extends·Omit<BootstrapButtonProps,·'size'·|·'variant'` with `·extends·Omit<⏎··BootstrapButtonProps,⏎··'size'·|·'variant'⏎`

(prettier/prettier)